vectors/cryptography_vectors/asymmetric/OpenSSH/ed448-nopsw.key.pub

```diff
@@ -0,0 +1 @@
+ssh-ed448 AAAACXNzaC1lZDQ0OAAAADnVY+2PC4Oj9MSsYZORD7xivKK3zyyHFKYj3eMCMPsAwNVk6fqGHeSIRDN39ld5Jto8S5Y1lemtJHA=
```

Comment:
Vectors should be documented in `test-vectors.rst`, thanks

Reply:
Thanks, done in 0ac431c.